kanidmd/src/lib/constants.rs

```diff
@@ -473,7 +473,6 @@ pub static JSON_IDM_ACP_XX_V1: &'static str = r#"{
         "name": ["idm_acp_xx"],
         "uuid": ["00000000-0000-0000-0000-ffffff0000XX"],
         "description": ["Builtin IDM Control for xx"],
-        "acp_enable": ["true"],
         "acp_receiver": [
```

Comment:
I think this should just be delted.

Reply:
Thanks for commenting! In a diff the '-' here means a removed line, and a '+' means a line was added. Similar the reason for the colour coding.

But good work to connect that (indeed) this attribute should be removed from the template :)